libs/components/code-examples/src/lib/modules/filter-bar/filter-bar/basic/filter-selection-values.ts

```diff
@@ -0,0 +1,81 @@
+import { SkyFilterBarFilterValue } from '@skyux/filter-bar';
+
+// eslint-disable @cspell/spellchecker
+export const FILTER_SELECTION_VALUES: Record<
+  string,
+  SkyFilterBarFilterValue[]
+> = {
+  'community-connection': [
+    { value: 'child-of-faculty', displayValue: 'Child of faculty' },
+    { value: 'child-of-alum', displayValue: 'Child of alum' },
+    { value: 'grandparent-is-alum', displayValue: 'Grandparent is an alum' },
+    { value: 'related-to-trustee', displayValue: 'Related to trustee' },
+    { value: 'sibling-candidate', displayValue: 'Sibling is a candidate' },
```

Comment:
```suggestion
    { value: 'sibling-candidate', displayValue: 'Sibling is candidate' },
```
We don't use articles in the other items so far, so I'd delete it here too and also in the items that follow. ... Alternatively, we could add the missing articles to the items before this.